pep-0007.txt

```diff
@@ -50,6 +46,11 @@ C dialect
   Future C99 features may be added to this list in the future
   depending on compiler support (mostly significantly MSVC).
 
+* Python versions before 3.6 use ANSI/ISO standard C (the 1989 version
```

Comment:
Perhaps past tense, as all of these are end of life. You could also make an argument for removing this stanza, as all of these are end of life.